acceptance/bundle/deploy/experimental-python/script

```diff
@@ -0,0 +1,3 @@
+trace uv run --quiet --python 3.12 --with databricks-bundles==0.7.3 -- $CLI bundle deploy
```

Comment:
I've seen some other code use Python 3.10 in acceptance tests (if I'm not mistaken).

I would prefer picking 1 version for all, or doing a version sweep for all, and not mix and match.

Reply:
Ok. I can set UV_PYTHON in test.toml closer to the root. We need to do the same for databricks-bundles==0.7.3